purchase_request_order_approved/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request Order Approved",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "version": "11.0.1.0.0",
+    "website": "http://www.eficent.com/",
```

Comment:
@hveficent Updated URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)